prometheus/README.md

```diff
@@ -0,0 +1,33 @@
+# Prometheus Monitoring - BOSH deployment details
+
+Prometheus is an Open source monitoring tool.  A BOSH release was created for it [prometheus-boshrelease](https://github.com/cloudfoundry-community/prometheus-boshrelease).
+
+The BOSH release deploys the following monitoring tools - [Prometheus](<https://prometheus.io/>), [AlertManager](https://github.com/prometheus/alertmanager) and  [Grafana](<https://grafana.com/>).
+
+
+## Usage
+
+### Setting up local BOSH environment
+The instructions for deploying a local BOSH2 director are [here](../setup-local-bosh.md)
+
+### Basic deployment
+The [prometheus boshrelease](https://github.com/cloudfoundry-community/prometheus-boshrelease) provides great documentation as well as the core BOSH deployment manifest and standard operator files.  
+This *DTA* repository provides some custom operator files and a concourse pipelines for deploying it to our environments.
+
+#### [aws-environment](operators/aws-environment.yml)
```

Comment:
There's a bit of duplication in this ops file and the other ops files. I think you intend that for cloud.gov.au we only use this one ops file, and the other ops files are for more specific customisations in other envs? Maybe rename this?